src/components/common/AsideNavbar/AsideNavbar.tsx

```diff
@@ -1,31 +1,33 @@
-import {useNavigate} from 'react-router';
-import {NavLink} from 'react-router-dom';
+import { useNavigate } from 'react-router';
+import { NavLink } from 'react-router-dom';
 import {
     BiTrash,
     BiPersonHearts,
     BiHouseHeart,
     BiBoxArrowRight,
-    BiArchive
+    BiArchive,
+    BiBoxArrowInLeft,
+    BiTags
 } from 'src/components/UI/Icons/Icons';
-import {useAuth} from 'src/context/AuthContext';
-import {useModal} from 'src/context/ModalProvider';
-import {ROUTE_PATH_ArchivePage, ROUTE_PATH_LandingPage, ROUTE_PATH_TrashPage} from 'src/utils/Route';
+import { useAuth } from 'src/context/AuthContext';
+import { useModal } from 'src/context/ModalProvider';
+import { ROUTE_PATH_ArchivePage, ROUTE_PATH_LabelPage, ROUTE_PATH_LandingPage, ROUTE_PATH_TrashPage } from 'src/utils/Route';
 import "./AsideNavbar.css";
 
 function AsideNavbar() {
     let auth = useAuth();
     let navigate = useNavigate();
     const { modalToggle, setmodalToggle } = useModal();
-    
-    const getActiveStyle = ({isActive}) => ({
-        color: isActive ? "var(--primary-color)" : "",
+
+    const getActiveStyle = ({ isActive }) => ({
+        color: isActive ? "var(--primary-color)" : "black",
         transform: isActive ? "scale(1.1)" : "",
         fontWeight: "700",
-        fontSize: "1.1em",
+        fontSize: "1em",
         cursor: "pointer",
         display: "flex",
         justifyContent: "flex-start",
-        gap: "1.5em",
+        gap: "0.5em",
```

Comment:
You can move this function outside of function component

Reply:
done